### PR TITLE
1269 - Content updates for Service Settings after login.gov integration

### PR DESCRIPTION
**Pull request by jonathanbobel**
## Description

Small content change for the Settings area after Login.gov goes live

### CODE REVIEW

**Review by terrazoon — COMMENTED** [review]
I think login.gov is always going to be enabled?  Why have this with a flag?

File: app/main/views/service_settings.py [hunk 1]
```diff
@@ -650,6 +650,7 @@ def service_set_channel(service_id, channel):
 def service_set_auth_type(service_id):
     return render_template(
         "views/service-settings/set-auth-type.html",
+        login_gov_enabled=False,
```
Comment (ccostino) [inline app/main/views/service_settings.py:653]: Wait, won't this change shut Login.gov integration off for the settings page?

**Review by ccostino — COMMENTED** [review]
One quick question, @jonathanbobel!

**Review by ccostino — APPROVED** [review]
Thanks, @jonathanbobel!

**Pull request merged**